added smoothing_timescale to boyd_pipe_operator

88hoddle opened the pull request:
Hi Steve, i have added smoothing_timescale to boyd_pipe_operator.

stoiver commented:
Hi Peter,

Your update is causing errors. Have a look at

https://travis-ci.org/GeoscienceAustralia/anuga_core/jobs/193580968

Probably just need to make a slight change to your calling arguments and 
push the changes to github.


Cheers

Steve


On 20/01/17 11:58, Petar Milevski wrote:
>
> Hi Steve, i have added smoothing_timescale to boyd_pipe_operator.
>
> ------------------------------------------------------------------------
>
>
>         You can view, comment on, or merge this pull request online at:
>
> https://github.com/GeoscienceAustralia/anuga_core/pull/117
>
>
>         Commit Summary
>
>   * Hi Steve, i have added blockage to weir orifice trapezoid operator.
>   * Add files via upload
>   * Add files via upload
>   * Add files via upload
>
>
>         File Changes
>
>   * *M* anuga/parallel/parallel_boyd_pipe_operator.py
>     <https://github.com/GeoscienceAustralia/anuga_core/pull/117/files#diff-0>
>     (70)
>   * *M* anuga/parallel/parallel_operator_factory.py
>     <https://github.com/GeoscienceAustralia/anuga_core/pull/117/files#diff-1>
>     (9)
>   * *M* anuga/parallel/parallel_weir_orifice_trapezoid_operator.py
>     <https://github.com/GeoscienceAustralia/anuga_core/pull/117/files#diff-2>
>     (9)
>   * *M* anuga/structures/boyd_pipe_operator.py
>     <https://github.com/GeoscienceAustralia/anuga_core/pull/117/files#diff-3>
>     (92)
>   * *M* anuga/structures/weir_orifice_trapezoid_operator.py
>     <https://github.com/GeoscienceAustralia/anuga_core/pull/117/files#diff-4>
>     (299)
>   * *A* boyd_pipe_operator.py
>     <https://github.com/GeoscienceAustralia/anuga_core/pull/117/files#diff-5>
>     (378)
>   * *A* parallel_boyd_pipe_operator.py
>     <https://github.com/GeoscienceAustralia/anuga_core/pull/117/files#diff-6>
>     (341)
>   * *A* parallel_operator_factory.py
>     <https://github.com/GeoscienceAustralia/anuga_core/pull/117/files#diff-7>
>     (971)
>
>
>         Patch Links:
>
>   * https://github.com/GeoscienceAustralia/anuga_core/pull/117.patch
>   * https://github.com/GeoscienceAustralia/anuga_core/pull/117.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/GeoscienceAustralia/anuga_core/pull/117>, or mute 
> the thread 
> <https://github.com/notifications/unsubscribe-auth/AFTlJNbykOAodvar351kXZdV6nph7wU9ks5rUAa0gaJpZM4LozDE>.
>

-- 
======================================
Stephen Roberts
Undergraduate Convenor
Mathematical Sciences Institute
Room 2005 P.A.P. Moran Building #26B
The Australian National University
Canberra, ACT 2601
AUSTRALIA
Ph: +61 2 61254445
CRICOS: 00120C

88hoddle commented:
Hi Steve. Yes I know trapezoid is not right. can you remove the pull
request?

But the smoothingredients timescale works. can you push that?
Regards

On 20 Jan 2017 12:05, "Stephen Roberts" <notifications@github.com> wrote:

> Hi Peter,
>
> Your update is causing errors. Have a look at
>
> https://travis-ci.org/GeoscienceAustralia/anuga_core/jobs/193580968
>
> Probably just need to make a slight change to your calling arguments and
> push the changes to github.
>
>
> Cheers
>
> Steve
>
>
> On 20/01/17 11:58, Petar Milevski wrote:
> >
> > Hi Steve, i have added smoothing_timescale to boyd_pipe_operator.
> >
> > ------------------------------------------------------------------------
> >
> >
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/GeoscienceAustralia/anuga_core/pull/117
> >
> >
> > Commit Summary
> >
> > * Hi Steve, i have added blockage to weir orifice trapezoid operator.
> > * Add files via upload
> > * Add files via upload
> > * Add files via upload
> >
> >
> > File Changes
> >
> > * *M* anuga/parallel/parallel_boyd_pipe_operator.py
> > <https://github.com/GeoscienceAustralia/anuga_core/pull/117/files#diff-0
> >
> > (70)
> > * *M* anuga/parallel/parallel_operator_factory.py
> > <https://github.com/GeoscienceAustralia/anuga_core/pull/117/files#diff-1
> >
> > (9)
> > * *M* anuga/parallel/parallel_weir_orifice_trapezoid_operator.py
> > <https://github.com/GeoscienceAustralia/anuga_core/pull/117/files#diff-2
> >
> > (9)
> > * *M* anuga/structures/boyd_pipe_operator.py
> > <https://github.com/GeoscienceAustralia/anuga_core/pull/117/files#diff-3
> >
> > (92)
> > * *M* anuga/structures/weir_orifice_trapezoid_operator.py
> > <https://github.com/GeoscienceAustralia/anuga_core/pull/117/files#diff-4
> >
> > (299)
> > * *A* boyd_pipe_operator.py
> > <https://github.com/GeoscienceAustralia/anuga_core/pull/117/files#diff-5
> >
> > (378)
> > * *A* parallel_boyd_pipe_operator.py
> > <https://github.com/GeoscienceAustralia/anuga_core/pull/117/files#diff-6
> >
> > (341)
> > * *A* parallel_operator_factory.py
> > <https://github.com/GeoscienceAustralia/anuga_core/pull/117/files#diff-7
> >
> > (971)
> >
> >
> > Patch Links:
> >
> > * https://github.com/GeoscienceAustralia/anuga_core/pull/117.patch
> > * https://github.com/GeoscienceAustralia/anuga_core/pull/117.diff
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/GeoscienceAustralia/anuga_core/pull/117>, or mute
> > the thread
> > <https://github.com/notifications/unsubscribe-auth/
> AFTlJNbykOAodvar351kXZdV6nph7wU9ks5rUAa0gaJpZM4LozDE>.
> >
>
> --
> ======================================
> Stephen Roberts
> Undergraduate Convenor
> Mathematical Sciences Institute
> Room 2005 P.A.P. Moran Building #26B
> The Australian National University
> Canberra, ACT 2601
> AUSTRALIA
> Ph: +61 2 61254445 <+61%202%206125%204445>
> CRICOS: 00120C
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GeoscienceAustralia/anuga_core/pull/117#issuecomment-273947829>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AScWthUpmHOGVY609n91ymdmYhUG-4Xcks5rUAhkgaJpZM4LozDE>
> .
>
